usr/share/rear/output/default/950_copy_result_files.sh

```diff
@@ -100,6 +100,7 @@ case "$scheme" in
         # see https://github.com/rear/rear/issues/1068
         LogPrint "Copying result files '${RESULT_FILES[*]}' to $scheme location"
         Log "lftp -c open $OUTPUT_URL; $OUTPUT_LFTP_OPTIONS mput ${RESULT_FILES[*]}"
+        lftp -c "open $OUTPUT_URL; mkdir -fp /$(echo $OUTPUT_URL | cut -d"/" -f4-)"
```

Comment:
I thik the hardcoded `cut -d"/" -f4-` could be dangerous because
it results an implicit condition how $OUTPUT_URL must look like.
I did not check the details elsewhere but from plain looking at this piece of code here
I cannot see and understand that we can rely on that $OUTPUT_URL
always has the right form that the hardcoded `cut -d"/" -f4-` results the right thing.
If can rely on the $OUTPUT_URL form, please add a comment that explains
why `cut -d"/" -f4-` always results the right thing that we need here.
Ideally add another local helper variable with a meansingful name that explains
what the $(echo $OUTPUT_URL | cut -d"/" -f4-) value actually is,
cf. "intermediate variables" in "Code must be easy to read" in
https://github.com/rear/rear/wiki/Coding-Style